kclvm/tools/src/LSP/src/find_refs.rs

```diff
@@ -13,46 +14,37 @@ use std::collections::HashMap;
 use std::sync::Arc;
 
 pub(crate) fn find_refs<F: Fn(String) -> Result<(), anyhow::Error>>(
-    program: &Program,
+    _program: &Program,
     kcl_pos: &KCLPos,
     include_declaration: bool,
-    prog_scope: &ProgramScope,
+    _prog_scope: &ProgramScope,
     word_index_map: Arc<RwLock<HashMap<Url, HashMap<String, Vec<Location>>>>>,
     vfs: Option<Arc<RwLock<Vfs>>>,
     logger: F,
+    gs: &GlobalState,
 ) -> Result<Vec<Location>, String> {
-    // find the definition at the position
-    let def = match program.pos_to_stmt(kcl_pos) {
-        Some(node) => match node.node {
-            Stmt::Import(_) => None,
-            _ => find_def(node.clone(), kcl_pos, prog_scope),
+   let def = find_def_with_gs(kcl_pos, &gs, true);
```

Comment:
Could U pls format this code to pass the CI check?

Reply:
ok

Reply:
> ok

The CI still fails. 
![image](https://github.com/kcl-lang/kcl/assets/22744597/a1b41eb7-3da6-4bbc-8544-d086e32e4b09)